### PR TITLE
Adding Alt tags to images

### PR DESCRIPTION
**Pull request by carrielaben-da**
Adding and improving alt tags to all images in the documents for accessibility. (Canton docs will be alt tagged separately.)

[CHANGELOG_BEGIN]

[CHANGELOG_END]

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/bindings-java/quickstart.rst [hunk 1]
```diff
@@ -163,6 +164,7 @@ Now everything is running, you can try out the quickstart application:
    This takes you to the contracts view:
 
    .. figure:: quickstart/images/contracts.png
+   :alt: The Contracts view in the application, showing one existing contract.
```
Comment (garyverhaegen-da) [inline docs/source/app-dev/bindings-java/quickstart.rst:167]:
```suggestion
      :alt: The Contracts view in the application, showing one existing contract.
```

File: docs/source/app-dev/bindings-java/quickstart.rst [hunk 2]
```diff
@@ -190,6 +192,7 @@ Now everything is running, you can try out the quickstart application:
 #. Try creating a contract from a template. Issue an Iou to yourself by clicking on the ``Iou:Iou`` row, filling it out as shown below and clicking **Submit**.
 
    .. figure:: quickstart/images/createIou.png
+   :alt: Fill out the form by entering Alice as the Issuer and as the Owner, AliceCoin as the Currency, and 1.0 as the Amount.
```
Comment (garyverhaegen-da) [inline docs/source/app-dev/bindings-java/quickstart.rst:195]:
```suggestion
      :alt: Fill out the form by entering Alice as the Issuer and as the Owner, AliceCoin as the Currency, and 1.0 as the Amount.
```

File: docs/source/app-dev/bindings-java/quickstart.rst [hunk 3]
```diff
@@ -209,6 +212,7 @@ Now everything is running, you can try out the quickstart application:
 #. To propose the trade, go to the **Templates** screen. Click on the *IouTrade:IouTrade* template, fill in the form as shown below and submit the transaction.
 
    .. figure:: quickstart/images/tradeProp.png
+   :alt: Fill out the form by entering Alice as the buyer, Bob as the seller, #13:1 as the baseIouCid, EUR_Bank as the baseIssuer, EUR as the baseCurrency, 100.00 as the baseAmount, USD_Bank as the quote_Issuer, USD as the quoteCurrency, and 110.0 as the quoteAmount.
```
Comment (garyverhaegen-da) [inline docs/source/app-dev/bindings-java/quickstart.rst:215]:
```suggestion
      :alt: Fill out the form by entering Alice as the buyer, Bob as the seller, #13:1 as the baseIouCid, EUR_Bank as the baseIssuer, EUR as the baseCurrency, 100.00 as the baseAmount, USD_Bank as the quote_Issuer, USD as the quoteCurrency, and 110.0 as the quoteAmount.
```

File: docs/source/concepts/ledger-model/ledger-exceptions.rst [hunk 4]
```diff
@@ -105,20 +107,23 @@ Here are three continuities implied by the "before-after" relation. The first:
...
    :width: 80%
+   :alt: The paint agreement flowchart with rollbacks, showing continuity for a scenario where Bank1 provides the Iou.
 
 The second:
 
 .. https://lucid.app/lucidchart/f1f92199-ae41-4de2-b1bf-0925d3ab89c9/edit
 .. image:: ./images/exception-integrity-continuity-2.svg
    :align: center
    :width: 80%
+    :alt: The paint agreement flowchart with rollbacks, showing continuity for a scenario where Bank2 provides the Iou.
```
Comment (garyverhaegen-da) [inline docs/source/concepts/ledger-model/ledger-exceptions.rst:118]:
```suggestion
   :alt: The paint agreement flowchart with rollbacks, showing continuity for a scenario where Bank2 provides the Iou.
```

File: docs/source/daml/daml-studio.rst [hunk 5]
```diff
@@ -139,6 +143,7 @@ created it.
 .. figure:: daml-studio/images/daml_studio_script_table_detailed.png
    :scale: 70%
    :align: center
+   :The table as described above, with the associated script in the left panel. "Show archived" and "Show detailed disclosure" are now selected.
```
Comment (garyverhaegen-da) [inline docs/source/daml/daml-studio.rst:146]:
```suggestion
   :alt: The table as described above, with the associated script in the left panel. "Show archived" and "Show detailed disclosure" are now selected.
```

File: docs/source/daml/intro/2_DamlScript.rst [hunk 6]
```diff
@@ -68,11 +68,12 @@ There are a few ways to run Daml Scripts:
...
 
 .. figure:: images/2_Scenario/scenario_results1.png
+   :alt: Script results indicating that a token has been created for Alice.
 
 This opens the script view in a separate column in VS Code. The default view is a tabular representation of the final state of the ledger:
 
 .. figure:: images/2_Scenario/tabular_view1.png
-
+:alt: The script view as a separate column, with a table that shows Alice's token. Full display explained immediately below.
 What this display means:
```
Comment (garyverhaegen-da) [inline docs/source/daml/intro/2_DamlScript.rst:77]:
```suggestion
   :alt: The script view as a separate column, with a table that shows Alice's token. Full display explained immediately below.

What this display means:
```

File: docs/source/daml/patterns/locking/locking-by-archiving.rst [hunk 7]
```diff
@@ -90,6 +91,7 @@ In the event that changing the original contract is not desirable and assuming t
   :end-before: -- END_COIN_COMMITMENT_TEMPLATE_LOCK_COIN
 
 .. figure:: ../images/lockingByArchiving2.png
+:alt: Locking by Archiving Contract uses the CoinCommitment contract to archive Coin and create LockedCoin
```
Comment (garyverhaegen-da) [inline docs/source/daml/patterns/locking/locking-by-archiving.rst:94]:
```suggestion
   :alt: Locking by Archiving Contract uses the CoinCommitment contract to archive Coin and create LockedCoin
```

File: docs/source/support/overview.rst [hunk 8]
```diff
@@ -22,9 +22,12 @@ Architecture
 A high level view of the architecture of a Daml application or solution is helpful to make sense of how individual components, APIs and features fit into the Daml Stack.
 
 .. figure:: architecture.png
+:alt: High level Daml architecture flowchart, described in-depth in the following two paragraphs.
```
Comment (garyverhaegen-da) [inline docs/source/support/overview.rst:25]:
```suggestion
   :alt: High level Daml architecture flowchart, described in-depth in the following two paragraphs.
```

File: docs/source/tools/navigator/index.rst [hunk 9]
```diff
@@ -303,6 +318,7 @@ If the access token cannot be retrieved, is missing or wrong, you'll be unable t
 .. image:: images/access-denied.png
   :width: 50%
   :align: center
+  :The Digital Asset Navigator home screen with the error message "You don't have the necessary authorization to access. Make sure to start the Navigator server with a valid access token." displayed in red.
```
Comment (garyverhaegen-da) [inline docs/source/tools/navigator/index.rst:321]:
```suggestion
  :alt: The Digital Asset Navigator home screen with the error message "You don't have the necessary authorization to access. Make sure to start the Navigator server with a valid access token." displayed in red.
```

File: docs/source/app-dev/app-arch.rst [hunk 10]
```diff
@@ -9,9 +9,12 @@ Application architecture
 This section describes our recommended design of a full-stack Daml application.
 
 .. image:: ./recommended_architecture.svg
+   :alt: Diagram of the recommended application architecture, explained in depth immediately below.
 
-The above image shows the recommended architecture. Of course there are many ways that the architecture and technology 
-stack can be changed to fit your needs which we'll mention in the corresponding sections.
+The above image shows the recommended architecture. Here there are four types of building blocks that go into our application: user code, Daml components, generated code from Daml, and external components. In the recommended architecture the Daml model determines the DAR files that underpin both the front end and back end. The front end includes user code such as a React Web Fronted, Daml React libraries or other integration libraries, and generated code from the DAR files. The back end consists of Daml integration components (e.g. JSON API) and a participant node; the participant node communicates with an external token issuer. The Daml network, meanwhile, includes Daml drivers paired with external synchronization technologies.
```
Comment (S11001001) [inline docs/source/app-dev/app-arch.rst:14]:
```suggestion
The above image shows the recommended architecture. Here there are four types of building blocks that go into our application: user code, Daml components, generated code from Daml, and external components. In the recommended architecture the Daml model determines the DAR files that underpin both the front-end and back-end. The front-end includes user code such as a React Web Frontend, Daml React libraries or other integration libraries, and generated code from the DAR files. The back-end consists of Daml integration components (e.g. JSON API) and a participant node; the participant node communicates with an external token issuer. The Daml network, meanwhile, includes Daml drivers paired with external synchronization technologies.
```

File: docs/source/app-dev/bindings-java/quickstart.rst [hunk 11]
```diff
@@ -163,6 +164,7 @@ Now everything is running, you can try out the quickstart application:
    This takes you to the contracts view:
 
    .. figure:: quickstart/images/contracts.png
+      :alt: The Contracts view in the application, showing one existing contract.
```
Comment (S11001001) [inline docs/source/app-dev/bindings-java/quickstart.rst:167]:
```suggestion
      :alt: The Contracts view in the Navigator, showing one existing contract.
```

File: docs/source/daml/patterns/locking/locking-by-safekeeping.rst [hunk 12]
```diff
@@ -42,6 +42,7 @@ There is no need to make a change to the original contract. With two additional
   :end-before: -- END_LOCKED_COIN_V2_TEMPLATE
 
 .. figure:: ../images/lockingBySafekeeping.png
+   :alt: Lockin by Safekeeping diagram, showing the ownership transfer of Coin and the creation of LockedCoin
```
Comment (S11001001) [inline docs/source/daml/patterns/locking/locking-by-safekeeping.rst:45]:
```suggestion
   :alt: Locking by Safekeeping diagram, showing the ownership transfer of Coin and the creation of LockedCoin
```

File: docs/source/daml/patterns/multiparty-agreement.rst [hunk 13]
```diff
@@ -71,5 +71,6 @@ Collecting the signatures in practice
 
 .. figure:: images/multiplepartyAgreement.png
   :figwidth: 80%
+  :alt: The Multipart Agreement pattern, in which the Pending contract recreates itself each time a party signs until all have signed and one exercizes the Finalize choice to create the Agreement contract.
```
Comment (S11001001) [inline docs/source/daml/patterns/multiparty-agreement.rst:74]:
```suggestion
  :alt: The Multiparty Agreement pattern, in which the Pending contract recreates itself each time a party signs until all have signed and one exercises the Finalize choice to create the Agreement contract. 
```

File: docs/source/getting-started/app-architecture.rst [hunk 14]
```diff
@@ -17,6 +17,9 @@ We generate TypeScript code to bridge the two.
 Overall, the social networking app is following the :ref:`recommended architecture of a fullstack Daml application <recommended-architecture>`. Below you can see a simplified version of the architecture represented in the app.
 
 .. image:: ./images/gsg_architecture.svg
+   :alt: An architectural diagram of the app from this guide. A full text explanation is found in the paragraph immediately below. 
+
+There are three types of building blocks that go into our application: user code, Daml components, and generated code from Daml. The Daml model determines the DAR files that underpin both the front end and back end. The front end includes React application code, Daml react libraries, and Typescript generated code, while the back end consists of a JSON API server and a participant node.
```
Comment (S11001001) [inline docs/source/getting-started/app-architecture.rst:22]:
```suggestion
There are three types of building blocks that go into our application: user code, Daml components, and generated code from Daml. The Daml model determines the DAR files that underpin both the front end and back end. The front-end includes React application code, Daml react libraries, and Typescript generated code, while the back-end consists of a JSON API server and a participant node.
```

File: docs/source/getting-started/testing.rst [hunk 15]
```diff
@@ -125,6 +125,7 @@ When writing CSS selectors for your tests, you will likely need to check the str
 For example, the image below is from inspecting the username field using the developer tools in Google Chrome.
 
    .. figure:: images/inspect-element.png
+      :alt: The app login screen and the code that renders as that screen. In the code a line that reads "<div class = "field test select username field> == $0" is highlighted.
```
Comment (S11001001) [inline docs/source/getting-started/testing.rst:128]:
```suggestion
      :alt: The app login screen and the code that renders as that screen. In the code a line that reads "<div class = "field test select username field"> == $0" is highlighted.
```

**Review by S11001001 — APPROVED** [review]
Looks good to me, modulo suggestions herein.

**Pull request merged**